src/unix/core.c

```diff
@@ -983,3 +984,26 @@ int uv__dup2_cloexec(int oldfd, int newfd) {
     return r;
   }
 }
+
+
+int uv_homedir(char* home, size_t len) {
```

Comment:
For consistency with other similar functions, len should be a pointer, which passes in the size of the buffer, and get written with the amount if written data minus the trailing null. Check uv_exepath for example.